grassroots-backend/package.json

```diff
@@ -5,36 +5,48 @@
   "author": "",
   "private": true,
   "license": "UNLICENSED",
+  "type": "module",
+  "exports": {
+    "./*": {
+      "require": "./dist-build/*.js",
+      "import": "./dist-build/*.js"
+    }
+  },
   "scripts": {
-    "postinstall": "if [ ! -f src/metadata.ts ]; then cp src/FormattedMetadata.gen.ts src/metadata.ts; fi",
-    "build": "nest build",
+    "build": "nest build && tsc -p tsconfig.buildwithtypes.json",
     "start": "nest start --watch",
-    "start:prod": "node dist/Main",
+    "start:prod": "node dist-build/Main",
     "gen-files-only": "nest start -- --gen-files-only",
-    "debug": "pnpm run build && node --inspect dist/Main",
+    "debug": "pnpm run build && node --inspect dist-build/Main",
+    "test:tstyche": "pnpm exec tstyche",
+    "db-create": "tsc -p tsconfig.mikroorm.json && pnpm exec mikro-orm migration:create --config dist/mikro-orm.config.js",
```

Comment:
The fact that the command needs to come before the "--config ..." is... terrible?

Otherwise I'd just create a single db command, and you could `pnpm run db -- migration:create`

We could create a wrapper script, but I think this is fine for now.

Reply:
Fixed this!